scanner/src/main/java/ca/on/oicr/gsi/runscanner/scanner/processor/RevioPacBioProcessor.java

```diff
@@ -350,8 +344,8 @@ private boolean isRunComplete(File runDirectory, int smrtCellCount) throws IOExc
    * Check for presence of Transfer_Test file and grab creation time
    *
    * @param runDirectory which we are currently processing
-   * @param dto PacBioNotificationDto which we will modify
-   *     <p>Throws NoSuchElementException if the .Transfer_Test files are missing
+   * @param dto PacBioNotificationDto which we will modify @Throws NoSuchElementException if the
```

Comment:
`@throws` needs to be lowercase and on its own line. If you hover over the method name in your IDE, it should pop up the JavaDoc, so you can see it's formatted correctly. Same on L375